.github/workflows/run-helm.yml

```diff
@@ -22,6 +28,11 @@ env:
 
 jobs:
   helm:
+    # Run workflow pull_request if it is NOT a fork, as pull_request_target if it IS a fork
+    if: >-
+      ( github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.fork == true ) ||
+      ( github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false ) ||
+      ( !(startsWith(github.event_name, 'pull_request')) )
```

Comment:
When does ` ( !(startsWith(github.event_name, 'pull_request')) )` occur?

Reply:
for a release

Reply:
or a push